libpod/container_internal_linux.go

```diff
@@ -668,11 +668,30 @@ func (c *Container) setupSystemd(mounts []spec.Mount, g generate.Generator) erro
 		}
 		g.AddMount(systemdMnt)
 	} else {
+		var statfs unix.Statfs_t
+		if err := unix.Statfs("/sys/fs/cgroup/systemd", &statfs); err != nil {
+			return errors.Wrapf(err, "/sys/fs/cgroup/systemd is not mounted on the host")
```

Comment:
if it is not mounted, could we just use a set of default mount options?

Reply:
Does that make sense given that we're trying to mount the same directory into the container? It seems like we ought to fail if we can't actually mount the cgroups in.